src/content/docs/setup/openid.mdoc

```diff
@@ -139,12 +142,14 @@ auth:
         clientid: <vikunja client-id>
         clientsecret: <vikunja client-secret>
         scope: openid profile email
+        forceuserinfo: false  # Optional: Set to true to always use UserInfo endpoint instead of ID token claims
       <provider-id2>:
         name: <provider2-name>
         authurl: <auth-url>  <----- Used for OIDC Discovery, usually the issuer
         clientid: <vikunja client-id>
         clientsecret: <vikunja client-secret>
         scope: openid profile email
+        forceuserinfo: false  # Optional: Set to true to always use UserInfo endpoint instead of ID token claims
```

Comment:
```suggestion
        forceuserinfo: false # Optional: Set to true to always use UserInfo endpoint instead of ID token claims, defaults to false
```